index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+      <header id="title">
+        <h1>nelson mandela</h1>
+        <p>the man who saved billions</p>
+      </header>
+      <div id="img-div">
+        <img id="image" src="<a title="©   copyright  John Mathew Smith  2001" href="https://commons.wikimedia.org/wiki/File:Nelson_Mandela_1994.jpg"><img width="256" alt="Nelson Mandela 1994" src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/02/Nelson_Mandela_1994.jpg/256px-Nelson_Mandela_1994.jpg"></a>" alt="photo of mandela">
+        <p id="img-caption">photo in color of Nelson mendela</p>
+        
+        <p>Nelson Mandela, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>
+      </div>
+      
+        </div>
+
+        <div id="tribute-info">
+            <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. 
+                Ipsum placeat magnam tenetur inventore asperiores quibusdam.
+                 Magnam amet exercitationem ducimus in, rerum voluptatem? 
+                 Id sit molestias, adipisci ea architecto quae dicta.</p>
+
+        <div>
+          <h1>
+            his <a id="tribute-link" href="https://en.wikipedia.org/wiki/Nelson_Mandela" target="_blank" >contribution</a> will know more and more.</h1>
+        </div>
+
+
+        <p>Nelson Mandela, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>
+      </div>
+      <div id="tribute-info">
+        <h3>Here's a time line of Dr. Borlaug's life:</h3>
+      </div>
+        <div class="time-line">
+          <p> .1914 - Born in Cresco, Iowa</p>
+          <p>1933 - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"</p>
+          <p>1935 - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."
+          </p>
+          <p>1937 - Finishes university and takes a job in the US Forestry Service</p>
+          <p>2009 - dies at the age of 95.</p>
+        </div>
```

Comment:
Your UI looks fine. However, I would suggest you to use 'section' tag instead of 'div' tag. Using semantic tag like 'section' increase accessibility of screen reader.